src/app/core/views/Preferences/tabs/Billing/CurrentPlanExtended.tsx

```diff
@@ -90,6 +90,10 @@ export default function CurrentPlanExtended({ className = '' }: { className?: st
     return plan.usageDetails?.total || -1;
   };
 
+  useEffect(() => {
+    dispatch(planThunks.initializeThunk()).unwrap();
+  }, [cancellingSubscription]);
```

Comment:
This will cause it to run several times during the cancellation process, when set to true and false. Also on line 48 this thunk is also called during cancellation in the function that cancels the subscription.
If it only has to be executed when the subscription is cancelled, wouldn't it be better to execute this thunk at the end of the cancellation function?

Reply:
I've tested it a couple of times with the 1 second setTimeout and it works.